src/rules/no_unreachable.rs

```diff
@@ -207,6 +207,28 @@ mod tests {
     assert_lint_ok::<NoUnreachable>("function foo() { try { a.b.c = 1; return; } catch (err) { return err; } }");
   }
 
+  #[test]
+  fn ok_10() {
+    assert_lint_ok::<NoUnreachable>(
+      r#"
+function normalize(type: string): string | undefined {
+  switch (type) {
+    case "urlencoded":
+      return "application/x-www-form-urlencoded";
+    case "multipart":
+      return "multipart/*";
+  }
```

Comment:
@kdy1 I think in situation where `switch` has no `default` it is non-exhaustive and shouldn't mark the block as "forced done".

Reply:
@bartlomieju  You are correct. Making it not forced-done will fix it.